packages/sdk/app-framework/src/plugins/common/navigation.ts

```diff
@@ -8,7 +8,7 @@ import type { IntentData } from '../IntentPlugin';
 import type { Plugin } from '../PluginHost';
 
 // NOTE(thure): These are chosen from RFC 1738’s `safe` characters: http://www.faqs.org/rfcs/rfc1738.html
-export const SLUG_LIST_SEPARATOR = '.';
+export const SLUG_LIST_SEPARATOR = '+';
```

Comment:
+ gives nicer URL semantics and allows plank id's to contain `.` like in the case of `dxos.org.plugin.debug.devtools`.

Reply:
@thure

Reply:
I'd be fine with this change, sure 👍